### PR TITLE
Debug Log level

### PR DESCRIPTION
**Pull request by lucasalcantara**
Addresses issue #124

### CODE REVIEW

**Comment by lucasalcantara** [comment]
@y0sher I've changed to Debug as much as I thought is ok do it. But one question, should I change literally all the Info to Debug or just that is fine?

**Comment by lucasalcantara** [comment]
@y0sher ok, so I will pass the rest to `Debug` thanks for the help and I will send the code in few minutes.

File: log/log.go [hunk 1]
```diff
@@ -18,32 +18,61 @@ type SpacemeshLogger struct {
...
-	logging.SetBackend(backendFormatter)
+	logFormat := ` %{color}%{level:.4s} %{id:03x} %{time:15:04:05.000} ▶%{color:reset} %{message}`
+	leveledBackend := getBackendLevel("app", "<APP>", logFormat)
+	log.SetBackend(leveledBackend)
 	smLogger = &SpacemeshLogger{Logger: log}
 }
 
+// getAllBackend returns level backends with an exception to Debug leve
+// which is only returned if the flag debug is set
+func getBackendLevel(module, prefix, format string) logging.LeveledBackend {
```
Comment (y0sher) [inline log/log.go]: So now prefix isn't used at all so we better just drop it.

**Comment by y0sher** [comment]
LGTM :+1:

**Pull request merged**